bears/r/FormatRBear.py

```diff
@@ -20,12 +20,12 @@ class FormatRBear:
 
     @staticmethod
     def create_arguments(filename, file, config_file,
-                         r_keep_comments: bool=True,
+                         r_keep_comments: bool=None,
```

Comment:
`tidy_source` already sets `r_keep_comments` to `true`.
If i want to effectively use the `dict` you gave below, then  `take_option` is false only when 
`r_keep_comments: bool=None` in the function declaration.